src/com/facebook/buck/cxx/CxxLibraryDescription.java

```diff
@@ -440,6 +440,7 @@ private static CxxLink createSharedLibrary(
       ImmutableSet<BuildTarget> blacklist,
       TransitiveCxxPreprocessorInputFunction transitiveCxxPreprocessorInputFunction,
       Optional<CxxLibraryDescriptionDelegate> delegate) {
+
```

Comment:
Looks like the code formatter you used added some whitespace. Please make sure to run Google's Java formatter on the PR (see Contributing.md)